src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationWithdrawn.cshtml

```diff
@@ -0,0 +1,42 @@
+﻿@using SFA.DAS.AdminService.Common.Extensions;
+@model SFA.DAS.ApplyService.Web.ViewModels.Roatp.ApplicationSummaryViewModel
+
+@{
+    ViewBag.Title = "Application withdrawn";
+    Layout = "_Layout";
+}
+
+<nav>
+    <ul id="navigation" class="roatp__navigation " aria-label="Top Level Navigation">
+        <li class="roatp__navigation-item">
+            <a class="govuk-link govuk-link--no-visited-state roatp__navigation-link--current">Application overview</a>
+        </li>
+        <li class="roatp__navigation-item roatp__navigation-item--signout">
+            <a class="govuk-link govuk-link--no-visited-state" asp-action="SignOut" asp-controller="Users">Save and sign out</a>
```

Comment:
It's part of the ticket but you can't Save anything on a Withdrawn application. Thus the content is incorrect for this application state. Can you query this with Design Team / Content.

Reply:
spoke to Dale. Have agreed the updated content and put it into a partial view